src/domain/entity/marketplaceCatalogItem.go

```diff
@@ -13,7 +13,9 @@ type MarketplaceCatalogItem struct {
 	Services           []valueObject.ServiceNameWithVersion          `json:"services"`
 	Mappings           []valueObject.MarketplaceItemMapping          `json:"mappings"`
 	DataFields         []valueObject.MarketplaceCatalogItemDataField `json:"dataFields"`
-	CmdSteps           []valueObject.MarketplaceItemCmdStep          `json:"-"`
+	InstallCmdSteps    []valueObject.MarketplaceItemCmdStep          `json:"-"`
+	UninstallCmdSteps  []valueObject.MarketplaceItemCmdStep          `json:"-"`
+	FilesToDelete      []valueObject.UnixFileName                    `json:"-"`
```

Comment:
This is loose here. FilesToDelete when? Why? Again, missing your intention, focusing on coding solely for it to work.

UninstallFilesToRemove, FilesToDeleteDuringUninstall  or similar would delivery your intention clearly, no questions asked.

Reply:
Adjusted!